int8 meta8
